scripts-available/CDB_ZoomFromScale.sql

```diff
@@ -1,10 +1,9 @@
 CREATE OR REPLACE FUNCTION cartodb.CDB_ZoomFromScale(scaleDenominator numeric) RETURNS int AS $$
```

Comment:
I think the current behaviour should be preserved for this function: Mapnik provides a `scale-denominator` value which is unique for all the tiles of a fixed zoom level, although the true scale will vary from tile to tile (with latitude). That is good for us, because we can unambiguosly derive the zoom level from that single scale value, so we can know which zoom level is being rendered with a given query that uses `!scale-denominator!`.

If we were to change this, having Mapnik provide the actual scale of each tile, we should either use a different token name, keeping the current behaviour of `scale-denominator` or add another token `zoom-level` which could be used in the cases where `CDB_ZoomFromScale` is now being used.